### DISCUSSION TITLE
is there any way to use web component from react to solidjs ?

### DISCUSSION THREAD

**Discussion by nemixe** [solidjs/solid#615] [Q&A]
in the last few days im playing around with package [direflow](https://direflow.io/), to convert react component into web component, as it turns out it will be good if i can use web component to solidjs as it stand now solidjs doesn't have much ui 
 tool library like react does, e.g: framer, chakra-ui, etc. 

so my question is, does solidjs already capable to implement custom web component from another framework ?

thanks you!.

**Comment by atk** [comment]
You can integrate web components seamlessly into your solid app, as it will render any DOM you throw at it. It even allows fine-grained control over whether props should be used as DOM attributes or properties.

**Reply by nemixe** [reply]
thanks for your reply, i tried it and success.
